Composer/packages/client/src/components/Orchestrator/DownloadNotification.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
+import { ProgressIndicator } from 'office-ui-fabric-react/lib/ProgressIndicator';
+
+import { CardProps } from '../Notifications/NotificationCard';
+
+const cardContent = css`
+  display: flex;
+  padding: 0 16px 16px 12px;
+  min-height: 64px;
+  align-items: center;
+`;
+
+const cardDetail = css`
+  margin-left: 8px;
+  flex-grow: 1;
+`;
+
+const infoType = css`
+  margin-top: 4px;
+  color: #0078d4;
```

Comment:
We can use a color from Fluent, I think this color is SharedColors.CyanBlue10

Reply:
Thanks @lei9444 , resolved.